packages/react-liveness/src/components/FaceLivenessDetector/service/utils/streamProvider.ts

```diff
@@ -117,7 +118,7 @@ export class LivenessStreamProvider extends AmazonAIInterpretPredictionsProvider
     const clientconfig: RekognitionStreamingClientConfig = {
       credentials,
       region: this.region,
-      customUserAgent: getAmplifyUserAgent(),
+      customUserAgent: `${getAmplifyUserAgent()} ${getLivenessUserAgent()}`,
```

Comment:
Could be way off base, but should we exclude the `customUserAgent` from the `clientConfig` if the credentials are non-cognito/custom? Just trying to confirm whether we need to consider anything related to custom credential providers here

Reply:
Don't think you're off base, I was actually considering whether we need the Amplify user agent at all? We only use the aws-amplify library for getting credentials in this case which is probably not very useful as debug information.

I can see multiple options here:
1. We only use the liveness ui package user agent string
2. We only include the Amplify JS user agent string if we use the default amplify credentials, if they pass the credential override we don't include it

What do you think?

Reply:
Ended up including both for now